### PR TITLE
Stop importing tutorials from k/examples repo

### PR DESCRIPTION
**Pull request by ahmetb**
This patch brings the tutorials back to the docs repo and removes full k8s.io
links + the import script. Once merged, I will update the README.md files in
k/examples repo with redirection notices.

/cc @jaredbhatti  @sebgoa @cody-clark
/assign @chenopis

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5086)

### CODE REVIEW

**Comment by sebgoa** [comment]
you guys decide, but the reason I don't like this is that I often go straight to the source of the tutorials to get some of the manifests or view the raw readme.

I used to go to kubernetes/kubernetes/examples we moved to kubernetes/examples

and now for guestbook I will need to go to kubernetes/docs/tutorials....and because of the docs rendering some of the raw markdown may not be complete.

**Comment by ahmetb** [comment]
@sebgoa this importing process has proven to be multiple times harder while contributing to the documentation. We had an intern revamping these tutorials and it took us 4-5 PRs to get the rendering right (because we would commit to k/examples and then import to the docs repository from there). It has been far too painful just to make little changes in the tutorials.

I think having these tutorials benefits neither the users nor the docs contributors. We can simply link out to the website for the tutorials. It seems like optimizing this for a few of us doesn't help the larger community and makes the contribution process more difficult.

File: docs/tutorials/stateful-application/cassandra.md [hunk 1]
```diff
@@ -67,20 +32,20 @@ The docker is based on `debian:jessie` and includes OpenJDK 8. This image includ
...
+To complete this tutorial, you should already have a basic familiarity with [Pods](/docs/concepts/workloads/pods/pod/), [Services](/docs/concepts/services-networking/service/), and [StatefulSets](/docs/concepts/workloads/controllers/statefulset/). In addition, you should:
 
-* [Install and Configure](https://kubernetes.io/docs/tasks/tools/install-kubectl/) the `kubectl` command line
+* [Install and Configure](/docs/tasks/tools/install-kubectl/) the `kubectl` command line
 
-* Download [cassandra-service.yaml](https://kubernetes.io/docs/tutorials/stateful-application/cassandra-service.yaml) and [cassandra-statefulset.yaml](https://kubernetes.io/docs/tutorials/stateful-application/cassandra-statefulset.yaml)
+* Download [cassandra-service.yaml](/docs/tutorials/stateful-application/cassandra-service.yaml) and [cassandra-statefulset.yaml](/docs/tutorials/stateful-application/cassandra-statefulset.yaml)
```
Comment (chenopis) [inline docs/tutorials/stateful-application/cassandra.md]: I'm getting a 404 for these links. Looks like they've been moved into the *cassandra* directory.

File: docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md [hunk 2]
```diff
@@ -64,19 +29,19 @@ A [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volu
...
 Download the following configuration files:
 
-1. [local-volumes.yaml](https://kubernetes.io/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/local-volumes.yaml)
+1. [local-volumes.yaml](/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/local-volumes.yaml)
 
-1. [mysql-deployment.yaml](https://kubernetes.io/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/mysql-deployment.yaml)
+1. [mysql-deployment.yaml](/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/mysql-deployment.yaml)
 
-1. [wordpress-deployment.yaml](https://kubernetes.io/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume//wordpress-deployment.yaml)
+1. [wordpress-deployment.yaml](/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume//wordpress-deployment.yaml)
```
Comment (ahmetb) [inline docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md]: fixed

File: docs/tutorials/stateful-application/cassandra.md (hunk 1 above)
Comment (ahmetb) [inline docs/tutorials/stateful-application/cassandra.md]: fixed

**Pull request merged**